### ISSUE TITLE
gist:Monetary should be a xsd:decimal

### ISSUE THREAD

**Issue by Jamie-SA** [semanticarts/gist#768]
Generally currency should use decimal numbers instead of floating point numbers (because of rounding and precision issues).
Add a further restriction on gist:Monetary: `gist:numericValue some xsd:decimal`.

**Comment by uscholdm** [comment]
This does make sense, on the face of it. It might be moot if we remove subclasses of Magnitude per #697.  

See also #703

**Comment by rjyounes** [comment]
@Jamie-SA @uscholdm Is there a general issue with our use of `xsd:decimal`, `xsd:double`, and `xsd:float`?

**Comment by Jamie-SA** [comment]
@rjyounes there are general issues with the use of different numeric datatypes for different types of data. I will discuss this during an upcoming OKE meeting.

But as Michael said, gist:Monetary is probably going to go away with the UoM changes making this issue request un-needed. Though I'm not sure it should be closed until the (soon to be released) UoM proposal is adopted.

**Comment by rjyounes** [comment]
Agreed, we should keep it open - though that change has not yet been approved by the gist development team.

**Comment by stevenchalem** [comment]
`gist:Monetary` no longer exists.
